README.md

```diff
@@ -1823,7 +1823,7 @@ Sets the list of resources to look for when a client requests an index of the di
 
 ##### `docroot`
 
-**Required**. Sets the [`DocumentRoot`][] location, from which Apache serves files.
```

Comment:
@roidelapluie somehow, i was hoping you'd fix it by making it **not** required anymore ;)